models/robots/schunk_pg70/eigen/eigen.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+<!-- d0: pg70_finger1_joint, min=-0.001, max=0.04 -->
+<!-- d1: pg70_finger2_joint, min=-0.001, max=0.04 -->
+<EigenGrasps dimensions="2">
+	<EG>
```

Comment:
hello, i'd like to know how to obtain these robot configuration file xml and iv? please help me?

Reply:
Hey @majiqiang, thanks for the interest. You can check you the Schunk PG70 files, here:
https://github.com/Roboskel-Manipulation/manos_graspit_config

Create an issue there if you need further assistance, so that we don't spam this repository.